onnx/defs/generator/defs.cc

```diff
@@ -111,6 +111,7 @@ ONNX_OPERATOR_SET_SCHEMA(
                     if (input_shape_dim_value > 0) {
                         for (int i = 0; i < input_shape_dim_value; ++i) {
                             auto newdim = final_output_shape->add_dim();
+                            (void)(newdim); // To eliminate "unused variable" compiler warning.
```

Comment:
Or just call the function, not creating the variable newdim to store the return value?